### PR TITLE
[fix](mow) cloud full compaction should check unique key when trying to calculate delete bitmap

### PR DESCRIPTION
**Pull request by hust-hhb**
## Proposed changes
now cloud full compaction on duplicate key will make be core, because it try to calculate delete bitmap which only exists in mow table.

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hust-hhb** [comment]
run buildall

File: be/src/cloud/cloud_full_compaction.cpp [hunk 1]
```diff
@@ -295,7 +295,14 @@ void CloudFullCompaction::do_lease() {
     }
 }
 
-Status CloudFullCompaction::_cloud_full_compaction_update_delete_bitmap(int64_t initiator) {
+Status CloudFullCompaction::_cloud_full_compaction_update_delete_bitmap_if_necessary(
+        int64_t initiator) {
+    if (_tablet->keys_type() != KeysType::UNIQUE_KEYS) {
```
Comment (zhannngchen) [inline be/src/cloud/cloud_full_compaction.cpp:300]: It would be more intuitive to put this judgment logic outside of the function call

Comment (hust-hhb) [reply be/src/cloud/cloud_full_compaction.cpp]: already change the code, take a look please @zhannngchen

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Pull request merged**